doc/content/the-things-stack/host/aws/ecs/mutual-tls/_index.md

```diff
@@ -71,49 +70,56 @@ If you want to set the certificates later, create an empty `index.yml` file.
 
 But since credentials once stored in ACM cannot be retrieved (can only be referenced by NLB), we need to store a copy of these credentials in the AWS Secrets Manager.
 
-Deploy the updated `4-1-secrets` template to create a new secret in ACM. This also generates additional keys in the KeyVault for {{% tts %}} components to exchange auth information.
+In `4-1-secrets` template with `SupportProxyTLS` set to `true`.
+
+This creates a new secret in ACM and also a `GCS token hash key secret` in the KeyVault for {{% tts %}} components to exchange auth information.
```

Comment:
I don't understand this

Reply:
I'm not sure what's not clear but I've reworded the sentence.